test/acceptor_SUITE.erl

```diff
@@ -702,6 +704,49 @@ ssl_sni_fail(_) ->
 	{'EXIT', _} = begin catch ranch:get_port(Name) end,
 	ok.
 
+ssl_tls_psk(_) ->
+	doc("Ensure that TLS-PSK works without certificate"),
+	Name = name(),
+	Ciphers = [#{cipher => aes_256_gcm, key_exchange => psk, mac => aead, prf => sha384}],
+	LookupFun = {fun psk_lookup_helper/3, <<"shared_secret">>},
+	{ok, _} = ranch:start_listener(Name,
+		ranch_ssl, [{protocol, tls}, {handshake, full}, {ciphers, Ciphers},
+					{user_lookup_fun, LookupFun}],
```

Comment:
@Elzor I looked into the failing tests, and it looks like you have to add `{versions, ['tlsv1.2']}` here in order to make them pass on OTP>=23.

Reply:
I will merge with this fixed. Thanks!